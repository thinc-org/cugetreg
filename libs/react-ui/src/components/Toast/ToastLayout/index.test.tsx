import { shallow } from 'enzyme'

describe('ToastLayout', () => {
  const mockChildren = <div>children</div>
  const mockAction = <div>action</div>

  it('Should render correctly', async () => {
    const { ToastLayout } = await import('.')
    const wrapper = shallow(<ToastLayout actions={mockAction}>{mockChildren}</ToastLayout>)

    expect(wrapper).toMatchSnapshot()
  })
})
