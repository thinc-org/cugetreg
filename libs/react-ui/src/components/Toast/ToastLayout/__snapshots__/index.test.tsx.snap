// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastLayout Should render correctly 1`] = `
<Styled(div)>
  <Styled(div)>
    <div>
      children
    </div>
  </Styled(div)>
  <div>
    action
  </div>
</Styled(div)>
`;
